source/remediation_runbooks/RevokeUnusedIAMUserCredentials.yaml

```diff
@@ -28,9 +28,9 @@ parameters:
     allowedPattern: ^[\w+=,.@_-]{1,128}$
   MaxCredentialUsageAge:
     type: String
-    description: (Required) Maximum number of days within which a credential must be used. The default value is 90 days.
+    description: (Required) Maximum number of days within which a credential must be used. The default value is 45 days.
     allowedPattern: ^(\b([0-9]|[1-8][0-9]|9[0-9]|[1-8][0-9]{2}|9[0-8][0-9]|99[0-9]|[1-8][0-9]{3}|9[0-8][0-9]{2}|99[0-8][0-9]|999[0-9]|10000)\b)$
-    default: "90"
+    default: "45"
```

Comment:
I don't think this is the right fix - the problem is that different controls use this remediation runbook, some of which have a limit of 90 days and I think the new one is the one that has a limit of 45 days. You'd want to change the value of the parameter passed to this runbook rather than the default. It looks like the following controls call this runbook:
- AFSBP IAM.8
- CIS 1.2.0 1.3
- CIS 1.4.0 1.12
- PCI PCI.IAM.7
- SC IAM.8
If all the rest are 90, you could just change the one passed by the controls that are not 90, which I think is just CIS 1.4.0 1.12.

Reply:
I see, I thought the idea was that all of these needed to be 45 so changing it at this level would do it all at once. Will change.